doc/prompt_task_map.json

```diff
@@ -0,0 +1,746 @@
+{
+    "commonsense_qa": [
+        {
+            "task": "task073_commonsenseqa_answer_generation",
+            "prompt": [
+                "most_suitable_answer",
+				"question_answering",
+				"question_to_answer_index"
+            ]
+        }
+    ],
+    "dream": [
+        {
+            "task": "task247_dream_answer_generation",
+            "prompt": [
+                "baseline",
+				"read_the_following_conversation_and_answer_the_question"
+            ]
+        }
+    ],
+    "quail": [
+        {
+            "task": "task887_quail_answer_generation",
+            "prompt": [
+                "context_description_question_text",
+				"context_question_description_text",
+				"description_context_question_text"
+            ]
+        }
+    ],
+    "quartz": [
+    ],
+    "social_i_qa": [
+        {
+            "task": "task384_socialiqa_question_classification",
+            "prompt": [
+				"Check if a random answer is valid or not"
+            ]
+        },
+        {
+            "task": "task580_socialiqa_answer_generation",
+            "prompt": [
+				"Show choices and generate answer",
+				"Show choices and generate index"
+            ]
+        }
+    ],
+    "wiqa": [
+    ],
+    "cosmos_qa": [
```

Comment:
I noticed that you included the question generation tasks for some QA datasets. This is good because I remember the original T0 also uses such prompts. But for some QA datasets, you didn't include the question generation task (e.g., `task023_cosmosqa_question_generation` can be included here for `cosmos_qa`). Can we add all of them if we already have them in our current data?

Reply:
I've only added the tasks that had an equivalent prompt. In this case, we have a question generation task, but they don't have a question generation prompt for `cosmos_qa`.